packages/taiko-client/driver/preconf_blocks/api.go

```diff
@@ -307,10 +316,20 @@ func (s *PreconfBlockAPIServer) GetStatus(c echo.Context) error {
 	s.lookaheadMutex.Lock()
 	defer s.lookaheadMutex.Unlock()
 
+	endOfSequencingMarkerReceived := false
+
+	if s.rpc.L1Beacon == nil {
+		_, ok := s.sequencingEndedForEpoch.Get(s.rpc.L1Beacon.CurrentEpoch())
+		if ok {
+			endOfSequencingMarkerReceived = true
+		}
+	}
+
 	return c.JSON(http.StatusOK, Status{
 		Lookahead:                     s.lookahead,
 		TotalCached:                   s.payloadsCache.getTotalCached(),
 		HighestUnsafeL2PayloadBlockID: s.highestUnsafeL2PayloadBlockID,
+		EndOfSequencingMarkerReceived: endOfSequencingMarkerReceived,
```

Comment:
Is there a subscription-based way to know when the previous sequencer has provided a `EndOfSequencing` mark? So far it seems we have to poll many times this endpoint to check if we can start sequencing earlier.

Reply:
no there isn't.. I can add for you and we can add it to the spec. I'll emit the end of sequencing mark and the epoch it's for